Sources/MapboxDirections/MapMatching/MapMatchingResponse.swift

```diff
@@ -50,6 +50,8 @@ extension MapMatchingResponse: Codable {
         
         tracepoints = try container.decodeIfPresent([Tracepoint?].self, forKey: .tracepoints)
         matches = try container.decodeIfPresent([Match].self, forKey: .matches)
-
+        matches?.enumerated().forEach {
+            $0.element.routeIndex = $0.offset
+        }
```

Comment:
This is fine, but generally it’s clearer to use a for loop unless we’re doing something particularly functional with the array:

```suggestion
        for (index, match) in matches?.enumerated() ?? [] {
            match.routeIndex = index
        }
```